### PR TITLE
Update README w/ Hex.pm badges

### PR DESCRIPTION
**Pull request by andrewvy**
Let's get started!

- Added in fancy Hex.pm badges from shields.io
- Shifted Installation to the top
- The example was using alias instead of import. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,53 @@
...
+  [applications: [:alchemy]]
+end
+```
+
 ## Perform some experiments
 
 Lets say that you have a controller that returns a list of users, and you want to change how that list of users is fetched from the database. Unit tests can help you but they can only account for examples that you've currently considered. Alchemy allows you to test your new code live in production.
 
+In these examples, the functions are imported from `Alchemy.Experiment`. If you want, you can use these functions without importing.
+
```
Comment (keathley) [inline README.md]: I'm not sure if we need this. It seems like most elixir-ists will know that we're importing the functions. But I could be wrong about that.

Comment (andrewvy) [inline README.md]: No worries, just thought it'd clarify. But since the example has the `import Alchemy.Experiment`, it could be redundant.

**Comment by andrewvy** [comment]
Updated this, removing the line discussed above. :)

**Comment by keathley** [comment]
Awesome! :+1:

**Pull request merged**